### DISCUSSION TITLE
What is the Best Way to Copy an ObjDict to Another ObjDict?

### DISCUSSION THREAD

**Discussion by briandowns** [Jason2605/Dictu#641] [General] [answered]
What is the Best Way to Copy an ObjDict to Another ObjDict?

**Comment by Jason2605** [comment]
There are the copy and deepCopy methods, but deepCopy will currently break if there are cyclic references so just be careful with that

https://dictu-lang.com/docs/collections/dictionaries/#copying-dictionaries

**Reply by briandowns** [reply]
Sorry, I mean in the C code.

**Reply by Jason2605** [reply, marked as answer]
Ah then it’ll just be using the copy function in copy.c 

https://github.com/dictu-lang/Dictu/blob/develop/src/vm/datatypes/copy.c#L5

**Reply by briandowns** [reply]
Wow, sorry. I can't believe I missed that. Thanks!

**Reply by Jason2605** [reply]
No worries!
